composer.json

```diff
@@ -18,23 +18,25 @@
     "require": {
         "php": "^5.3.3|^7.0",
```

Comment:
You're dropping the 5.3 build in .travis.yml, so you should also drop support for 5.3 here. I would also suggest dropping PHP 5.x completely, but that depends on what the maintainers here want to maintain.